examples/training/classification/imagenet/basic_training.py

```diff
@@ -188,8 +188,9 @@ def augment(img, label):
 """
 
 
-optimizer = optimizers.SGD(learning_rate=INITIAL_LEARNING_RATE, momentum=0.9)
-
+optimizer = optimizers.SGD(
+    learning_rate=INITIAL_LEARNING_RATE, momentum=0.9, global_clipnorm=10
```

Comment:
Does this help you to stabilize the training?

Reply:
Yes, for a few models I've seen some wild swings in validation accuracy which were resolved with this clipping.

Reply:
Maybe this would be where the warmup lr schedule I mentioned earlier would help